jenkins/scripts/common/getNode.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash -ex
```

Comment:
So why not just use `nvm` like https://github.com/nodejs/build/blob/1fb7b6046ed7304fef291133a6599decb06243c7/jenkins/pipelines/node-inspect.jenkinsfile#L72-L78

Reply:
Does it work on windows?

Reply:
I don't have an issue with using nvm if it works everywhere and doesn't write to files outside the workspace.

This is what was there already, I just cleaned up the `sort`, which was a bit broken.

Reply:
llnode is not supported on Windows yet (as per https://github.com/nodejs/llnode/issues/17), so I don't think nvm not running on Windows is an issue now. When we get support to Windows, we can use `nvm` for Unix platforms and `nvs` for Windows (or just use `nvs` for everything).

But we can change to `nvm`/`nvs` in the future.